webpack.conf.js

```diff
@@ -0,0 +1,27 @@
+var webpack = require('webpack');
+
+module.exports = {
+    devtool: "inline-source-map",
+    bail: true,
+    module: {
+        loaders: [
+            {
+                test: /\.js$/,
+                loader: "babel-loader",
+                exclude: /node_modules/,
+                query: {
+                    compact: false,
+                    cacheDirectory: true
+                }
+            }
+        ]
+    },
+    plugins: [
+        new webpack.ProvidePlugin({
+            $: "jquery",
+            jQuery: "jquery",
+            'window.jQuery': 'jquery'
+        }),
+        new webpack.optimize.DedupePlugin()
```

Comment:
Cool plugin :) Do we want to only run this for prod build? In the doc (https://webpack.github.io/docs/list-of-plugins.html), it says 'Note: Don’t use it in watch mode. Only for production builds'. Probably because it slows down the build in watch mode?